src/exchange/passthrough.go

```diff
@@ -38,6 +40,11 @@ func NewPassthrough(log logrus.FieldLogger, cfg config.SkyExchanger, store Store
 		return nil, err
 	}
 
+	orderbookDatabase, err := db.NewOrderbookTracker()
```

Comment:
Future concern: Is the orderbook tracker necessary? Can't we query the orderbook each time?

Reply:
Yeah, I think you mentioned this before. It might not be necessary, or at the very least, we can add a construction function (instead of creating the `c2cx.Client` manually) and hide the db creation there.

If you make an issue on exchange-api and assign it to me, I can do the refactoring.